hudi-common/src/main/java/org/apache/hudi/common/util/BaseFileUtils.java

```diff
@@ -67,6 +70,50 @@ public static BaseFileUtils getInstance(HoodieFileFormat fileFormat) {
     throw new UnsupportedOperationException(fileFormat.name() + " format not supported yet.");
   }
 
+  /**
+   * Aggregate column range statistics across files in a partition.
+   *
+   * @param fileRanges List of column range statistics for each file in a partition
+   */
+  public static <T extends Comparable<T>> HoodieColumnRangeMetadata<T> getColumnRangeInPartition(@Nonnull List<HoodieColumnRangeMetadata<T>> fileRanges) {
+    if (fileRanges.size() == 1) {
```

Comment:
I think this is column range, and not multiple files? i.e list has an entry for each column for a single file. no?

Reply:
Yes, fixed the logic and added ut in `testGetColumnRangeInPartition`